src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -98,6 +114,8 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
     isGreetingMessage = false;
     shouldLoadGreetingMessage = true;
     fadeState = '';
+    exerciseId: number;
+    sessionService: IrisSessionService;
```

Comment:
Why don't you use dependency injection like for the rest of the services?

Reply:
inject `IrisSessionService` as a result will inject `IrisStateStore` (it would create a new instance of `IrisStateStore`), but I need to use the `IrisStateStore` that has already been injected in `src/main/webapp/app/iris/exercise-chatbot/exercise-chatbot.component.ts` so we can have the same state